styleguide/webpack-styleguide-config.js

```diff
@@ -4,18 +4,13 @@ const webpack = require('webpack');
 const { JustSsrPlugin } = require('vue-just-ssr');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const ent = require('ent');
-const branch = require('git-branch');
 const { merge } = require('../build/utils');
 const webpackBaseConfig = require('../build/webpack-base-config');
 const { version } = require('../package.json');
+const { getCurrentBranch, isSemanticReleaseBranch, getLibVersion } = require('../build/utils');
 
-// branch.sync() doesn't work in the Github Actions env
-let branchName = branch.sync();
-if (!branchName) {
-	// but that's okay because the Github Actions env provides this var
-	branchName = process.env.GITHUB_HEAD_REF;
-}
-const deploy = branchName === 'master' ? version : branchName;
+let branchName = getCurrentBranch();
+const deploy = isSemanticReleaseBranch(branchName) ? getLibVersion() : branchName;
```

Comment:
My understanding of `getLibVersion()` is that it's specifically for post semantic-release CI.

I'm curious if doing that is necessary--does `semantic-release` actually revert the version bump in `package.json`? I'm thinking not since it's designed for CI and it expects changes to be scrapped.

Reply:
`getLibVersion()` is generally useful. I used it to checkout old commits and build & deploy the styleguides for versions: v3.0.0, v4.0.0, v4.0.1, v4.1.0, v4.1.1, v4.1.2, v4.1.3, and v4.1.4 which are all now live on the [public deploy index](https://square.github.io/maker/). It's nice to have a script that works when run locally in case CI ever fails, or in case we ever want do some one-off styleguide deploy independent of CI for whatever reason. I'd like to keep `getLibVersion()` as-is because it works locally.

Reply:
FYI `npx semantic-release` will work locally too if you want to release yourself